lib/ts/string_view.h

```diff
@@ -1224,7 +1224,7 @@ template <class _Type, class _Traits> struct hash<ts::basic_string_view<_Type, _
 // This should also mean we have some consistent behavior with std code
 #if defined(__linux__)
     return std::_Hash_impl::hash(x.data(), x.length() * sizeof(typename string_type::value_type));
-#elif defined(__FreeBSD__)
+#elif defined(__FreeBSD__) || defined(__APPLE__)
```

Comment:
Seems like we should standardize on our ifdefs for platforms.  I mostly see this in our code:
`#if defined(freebsd) || defined(darwin)`

Reply:
I agree .. I did not see the Darwin define.. I will fix